frontend/src/interfaces/route.ts

```diff
@@ -29,8 +29,7 @@ export interface INavigationMenuItem {
 }
 
 interface IRouteMenu {
-    mobile?: boolean;
-    advanced?: boolean;
+    main?: boolean;
```

Comment:
There's not 'advanced' menu anymore. Renaming to 'main' - as that's where it ends up in downstream properties